lib/bcp47/registry.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require 'active_support'
+require 'active_support/core_ext/object/blank'
+require 'active_support/core_ext/string/inflections'
+require 'faraday'
+
+module Bcp47
+  class Registry
+    CODE_DELIMITER = '-'
+    LABEL_DELIMITER = ', '
+    LANGUAGE_SUBTAG_REGISTRY_URL = 'https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry'
+
+    def self.records
+      @records ||= Parser.new(Faraday.get(LANGUAGE_SUBTAG_REGISTRY_URL).body).records
```

Comment:
Can we split out the download part?  It seems better if we don't have to depend on a remote service.  Perhaps this could be a rake task to update the file that you keep in the repo.

Reply:
Yes, that sounds good. I'll stick the file somewhere in `app`, I guess, and toss a simple rake task at the PR

Reply:
@jcoyne Done.